runtime/doc/syntax.txt

```diff
@@ -4864,7 +4864,7 @@ SignColumn	column where |signs| are displayed
 							*hl-IncSearch*
 IncSearch	'incsearch' highlighting; also used for the text replaced with
 		":s///c"
-							*hl-Substitute*
+						 {Nvim} *hl-Substitute*
```

Comment:
Should I remove them in those commit or just close the PR and we'll do it another time when `doc/syntax.txt` gets altered?